src/nfv_test_api/v2/controllers/gnodeb.py

```diff
@@ -108,7 +108,9 @@ def __init__(self, api=None, *args, **kwargs):
     @namespace.response(
         HTTPStatus.OK, "Found a gNodeB config with a matching nci", gnodeb_model
```

Comment:
```suggestion
        HTTPStatus.OK.value, "Found a gNodeB config with a matching nci", gnodeb_model
```
Did you check that none was missing? I can see a few ones that still need to be addressed

Reply:
I used your expression to find them, I will have a look again.

Reply:
Not a few ones actually, just that one, but I might have missed some

Reply:
Actually I missed most of them, this is because of the formatting your expression doesn't match them (new line, ...).